hela/posteriors.py

```diff
@@ -179,6 +180,46 @@ def __init__(self, samples, weights):
         self.samples = samples
         self.weights = weights
 
+    def print_percentiles(self, names):
+        """
+        Print the median and the +/- 1 sigma posterior values.
+
+        Parameters
+        ----------
+        names
+
+        Returns
+        -------
+
+        """
+        posterior_ranges = self.data_ranges()
+        for name_i, pred_range_i in zip(names, posterior_ranges):
+            print("Prediction for {}: {:.3g} "
+                  "[+{:.3g} -{:.3g}]".format(name_i, *pred_range_i))
+
+    def data_ranges(self, percentiles=(50, 16, 84)):
+        """
```

Comment:
This method is fine, but it seems a bit strange that this method is in the class `Posterior` and the function `resample_posterior` isn't. Rather put them both as separated functions or both as methods of this class? I have a slight preference towards independent functions and against classes, but if you prefer methods it'll be fine for me.